srcpkgs/xbps-triggers/files/dkms

```diff
@@ -21,19 +21,24 @@ UPDATE="$5"
 export PATH="usr/bin:usr/sbin:/usr/sbin:/usr/bin:/sbin:/bin"
 
 remove_modules() {
-	local _modver _kver
-
 	# Remove the specified modules from all kernels.
 	set -- ${dkms_modules}
 	while [ $# -gt 0 ]; do
-		$DKMS status -m "$1" | while read line; do
-			if $(echo "$line" | egrep -vq '(added|built|installed)'); then
-				shift 2; continue
+		$DKMS status -m "$1" | while read -r line; do
+			IFS=,:/ read -r modname modver kver arch status _ <<-EOF
```

Comment:
On further testing, it turns out `IFS=,:/` is not sufficient because it doesn't recognize whitespace as a separator. As a consequence, `$kver`, `$arch` and `$status` all contain leading spaces. That causes the test on line 35 to fail and modules are never removed when the package is removed.

Try, for example, with this version of `xbps-triggers` and `dkms` installed along with one or more kernels:
```sh
dkms status
xbps-install nvidia-dkms
dkms status
xbps-remove nvidia-dkms
dkms status
```

The output of `xbps-remove` will show `Removing nvidia-dkms-515.57_1... ` and regenerate your initramfs images, but it will not print any messages about removing the module for specific kernels. The final `dkms status` will report an error and provide an incomplete listing of installed modules because it chokes on `nvidia` after the package files are removed.